dev/com.ibm.ws.concurrent_fat_config/fat/src/com/ibm/ws/concurrent/fat/EEConcurrencyConfigTest.java

```diff
@@ -441,6 +441,7 @@ public void testManagedThreadFactoryUsableAcrossAppRestart() throws Exception {
         server.setMarkToEndOfLog();
         server.updateServerConfiguration(config);
         server.waitForConfigUpdateInLogUsingMark(Collections.emptySet());
+        server.waitForStringInTraceUsingMark("CWWKZ0009I"); //CWWKZ0009I: The application concurrent has stopped successfully.
```

Comment:
Nice work debugging the cause and identifying where to correct it.
Instead of scanning trace files for the message, you should be able to have `waitForConfigUpdateInLogUsingMark(Collections.emptySet(), "CWWKZ0009I")` wait for it in the messages file, which is much shorter than trace,

Reply:
Thanks for the suggestion.  I forgot you could add messages to the end of the config update.